tests/components/miele/snapshots/test_climate.ambr

```diff
@@ -255,3 +255,195 @@
+    'original_icon': None,
+    'original_name': 'Refrigerator',
+    'platform': 'miele',
+    'previous_unique_id': None,
+    'suggested_object_id': None,
+    'supported_features': <ClimateEntityFeature: 1>,
+    'translation_key': 'refrigerator',
+    'unique_id': 'DummyAppliance_Fridge_Freezer-thermostat-1',
+    'unit_of_measurement': None,
+  })
+# ---
+# name: test_climate_states_mulizone[fridge_freezer-fridge_freezer.json-platforms0][climate.fridge_freezer_refrigerator-state]
+  StateSnapshot({
+    'attributes': ReadOnlyDict({
+      'current_temperature': 4,
+      'friendly_name': 'Fridge freezer Refrigerator',
+      'hvac_modes': list([
+        <HVACMode.COOL: 'cool'>,
+      ]),
+      'max_temp': 9,
+      'min_temp': 1,
+      'supported_features': <ClimateEntityFeature: 1>,
+      'target_temp_step': 1.0,
+      'temperature': 4,
+    }),
+    'context': <ANY>,
+    'entity_id': 'climate.fridge_freezer_refrigerator',
+    'last_changed': <ANY>,
+    'last_reported': <ANY>,
+    'last_updated': <ANY>,
+    'state': 'cool',
+  })
+# ---
+# name: test_climate_states_mulizone[fridge_freezer-fridge_freezer.json-platforms0][climate.fridge_freezer_zone_3-entry]
+  EntityRegistryEntrySnapshot({
+    'aliases': set({
+    }),
+    'area_id': None,
+    'capabilities': dict({
+      'hvac_modes': list([
+        <HVACMode.COOL: 'cool'>,
+      ]),
+      'max_temp': -15,
+      'min_temp': -30,
+      'target_temp_step': 1.0,
+    }),
+    'config_entry_id': <ANY>,
+    'config_subentry_id': <ANY>,
+    'device_class': None,
+    'device_id': <ANY>,
+    'disabled_by': None,
+    'domain': 'climate',
+    'entity_category': None,
+    'entity_id': 'climate.fridge_freezer_zone_3',
+    'has_entity_name': True,
+    'hidden_by': None,
+    'icon': None,
+    'id': <ANY>,
+    'labels': set({
+    }),
+    'name': None,
+    'options': dict({
+    }),
+    'original_device_class': None,
+    'original_icon': None,
+    'original_name': 'Zone 3',
+    'platform': 'miele',
+    'previous_unique_id': None,
+    'suggested_object_id': None,
+    'supported_features': <ClimateEntityFeature: 1>,
+    'translation_key': 'zone_3',
+    'unique_id': 'DummyAppliance_Fridge_Freezer-thermostat3-3',
+    'unit_of_measurement': None,
+  })
+# ---
+# name: test_climate_states_mulizone[fridge_freezer-fridge_freezer.json-platforms0][climate.fridge_freezer_zone_3-state]
+  StateSnapshot({
+    'attributes': ReadOnlyDict({
+      'current_temperature': -28,
+      'friendly_name': 'Fridge freezer Zone 3',
+      'hvac_modes': list([
+        <HVACMode.COOL: 'cool'>,
+      ]),
+      'max_temp': -15,
+      'min_temp': -30,
+      'supported_features': <ClimateEntityFeature: 1>,
+      'target_temp_step': 1.0,
+      'temperature': -25,
+    }),
+    'context': <ANY>,
+    'entity_id': 'climate.fridge_freezer_zone_3',
+    'last_changed': <ANY>,
+    'last_reported': <ANY>,
+    'last_updated': <ANY>,
+    'state': 'cool',
+  })
+# ---
```

Comment:
We should not do freezer or fridges as climate entities. This doesn't work for auto generated dashboards or with voice control. This could cause people to set their devices to an unhealthy temperature without knowing. (not a blocker btw, but food for thought)

Reply:
You are touching an interesting topic. I cannot see any problems with the auto generated dashboard for my kitchen. However, I can see problems with voice control being too permissive. It is not specific to freezers and fridges though. Imagine you say "turn off bathroom" and it includes not only lights and media players but also heating, water heater etc. In middle of the winter in Sweden with -20C outdoors!

But that is perhaps not a discussion for this PR.